src/trigger.py

```diff
@@ -38,6 +38,9 @@ def _run_trigger(args, build_config, db):
     else:
         node_list = get_node_by_commit_id(
             kernelci.build.get_branch_head(build_config))
+        if len(node_list) != 0:
```

Comment:
This is correct, but `if not node_list:` is a bit better for checking whether a collection is empty.  It's something `pylint` would generate a warning about.